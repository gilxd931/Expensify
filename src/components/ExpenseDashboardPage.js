import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpanseDashboardPage = () => {
    return (
        <div>
            <ExpensesSummary />
            <ExpenseListFilters />
            <ExpenseList />
        </div>
    );
};

export default ExpanseDashboardPage;
