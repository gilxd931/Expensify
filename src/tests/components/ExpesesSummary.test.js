import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';


test('render expesnes summary with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});


test('render expesnes summary with multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={23} expensesTotal={3241212} />);
    expect(wrapper).toMatchSnapshot();

});