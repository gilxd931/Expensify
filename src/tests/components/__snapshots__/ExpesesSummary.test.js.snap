// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render expesnes summary with 1 expense 1`] = `
<div>
  <h1>
    view 
    1
     
    expense
     total of 
    $2.35
  </h1>
</div>
`;

exports[`render expesnes summary with multiple expense 1`] = `
<div>
  <h1>
    view 
    23
     
    expenses
     total of 
    $32,412.12
  </h1>
</div>
`;
